components/todo/TodoItem.jsx

```diff
@@ -3,7 +3,14 @@ import styled from "styled-components";
 import Image from "next/image";
 import deleteIcon from "../../public/icons/delete.svg";
 
-export const TodoItem = ({ name, id, isDone, deleteTodo, toggleClick }) => {
+export const TodoItem = ({
+  name,
+  id,
+  isDone,
+  deleteTodo,
+  toggleClick,
+  editAble = true,
```

Comment:
maybe `isEditable` to make clear that it's a boolean

Reply:
check